mrqacheck/audit.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3
```

Comment:
shebang so can call from command line like `mrqacheck/audit.py` 
also ran `chmod +x mrqacheck/audit.py` so it's executable (permission bits tracked by git too)